core/cap-0035.md

```diff
@@ -463,8 +463,8 @@ operation.
 
 Possible return values for the `ClawbackClaimableBalanceOp` are:
 - `CLAWBACK_CLAIMABLE_BALANCE_SUCCESS` if the clawback is successful.
-- `CLAWBACK_CLAIMABLE_BALANCE_MALFORMED` if the `claimableBalanceId` value is
-  malformed.
+- `CLAWBACK_CLAIMABLE_BALANCE_DOES_NOT_EXIST` if the claimable balance does not
+exist.

```

Comment:
I think this change is good. It's consistent with the `ClaimClaimableBalanceOp`.

Could you format it consistently with the document by wrapping to 80 and indenting the next line three characters?

Reply:
f33975e. Turns out the way you were doing it works and all the indenting we had throughout was unnecessary so I updated all other places to match. Thanks!